File: app/src/main/java/com/ytempest/wanandroid/base/presenter/IPresenter.java
package com.ytempest.wanandroid.base.presenter;

import com.ytempest.wanandroid.base.view.IView;

/**
 * @author heqidu
 * @since 2020/6/28
 */
public interface IPresenter {

    <V extends IView> void attachView(V view);

    void detach();
}

File: app/src/main/java/com/ytempest/wanandroid/base/presenter/IPresenter.kt
package com.ytempest.wanandroid.base.presenter

import com.ytempest.wanandroid.base.view.IView

/**
 * @author heqidu
 * @since 21-2-7
 */
interface IPresenter {

    fun <V : IView> attachView(view: V)

    fun detach()
}